db/migrate/20170613144708_add_sti_to_container_image.rb

```diff
@@ -0,0 +1,20 @@
+class AddStiToContainerImage < ActiveRecord::Migration[5.0]
+  class ContainerImage < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    add_column :container_images, :type, :string
```

Comment:
Is it possible to create one migration for structural changes and one for data changes? (you can use `change` method in structural migration)